src/Fls.Results/OperationResult.cs

```diff
@@ -128,36 +234,64 @@ public static IOperationResult<T> BindError<T>(this IOperationResult<T> source,
             );
         }
 
+        /// <summary>
+        /// Async version of Bind, to suport functional pipeline with async Task as output of previous operation
+        /// </summary>
+        /// <returns>Task from one of value type SuccessResult, ErrorResult or FailureResult</returns>
         public static async Task<IOperationResult<TOut>> BindAsync<TIn, TOut>(this Task<IOperationResult<TIn>> source, Func<TIn, Task<IOperationResult<TOut>>> bindAsync)
         {
             return await (await source).BindAsync(bindAsync);
         }
 
+        /// <summary>
+        /// Async version of Bind, to suport functional pipeline with async Task as output of previous operation
+        /// </summary>
+        /// <returns>Task from one of value type SuccessResult, ErrorResult or FailureResult</returns>
         public static async Task<IOperationResult<TOut>> BindAsync<TIn, TOut>(this Task<IOperationResult<TIn>> source, Func<TIn, IOperationResult<TOut>> bind)
         {
             return (await source).Bind(bind);
         }
 
+        /// <summary>
+        /// Async version of BindError
```

Comment:
This isn't helping. Cause when you reach out for help and point to the function in your code, the text 'async version of something else' might as well be something you don't want to see.